tests/src/Functional/RdfEntityTypeCreation.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Drupal\Tests\rdf_entity\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the creation of RdfEntityType entities (RDF bundles)
+ *
+ * @group rdf_entity
+ */
+class RdfEntityTypeCreation extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
```

Comment:
It's `protected`, no?

Reply:
@claudiu-cristea No, it's public. At least that is what's in other core tests.

Reply:
@upchuk You should check the parent class property, not other tests. The core is scheduled to correct this in https://www.drupal.org/project/drupal/issues/2822382

Reply:
@claudiu-cristea Ah yes, you are right. Thanks!